README.Rmd

```diff
@@ -159,3 +159,39 @@ By contributing to this project, you agree to abide by its terms.
 ```{r message=FALSE, warning=FALSE}
 citation("simulist")
 ```
+
+## Related projects
```

Comment:
In the original simulist project idea I had also listed [ringbp](https://github.com/epiforecasts/ringbp) as an example of a related project since you can also use that package to simulate outbreak data with contact tracing- this however doesn't give you a linelist, although the output could be used to generate a linelist as well, this could be mentioned in this section to highlight why simulist is useful

Reply:
I was going to add them both but while I was doing so I started to question whether they're actually relevant enough to add them to the table since they're not able to simulate linelists, which is the main point of simulist... what do you think? maybe they could be mentioned in a separate bullet point to say that there are other packages that might do stuff relevant for outbreak simulation but not necessarily provide tailored linelists to the user

Reply:
Thanks for the suggestion, I've added a few bullet points below what was already added to the Related projects section linking to {outbreak}, {ringbp} and {epichains} in 8e4370342e0fefb4225667581fcf04176fca350a.